controller/Dockerfile

```diff
@@ -3,9 +3,6 @@ FROM alpine:3.2
 # install common packages
 RUN apk add --update-cache curl bash sudo && rm -rf /var/cache/apk/*
 
-# install etcdctl
-RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/get-deis/etcdctl-v0.4.9 \
-    && chmod +x /usr/local/bin/etcdctl
 
```

Comment:
Probably check the router stuff assuming after the Microsoft buy that bucket is dead too?

https://github.com/Hearst-Hatchery/deis/blob/master/router/Dockerfile#L15

(I’ll test it Monday when we do the other change from Robert)